src/components/App.js

```diff
@@ -23,6 +22,17 @@ export default class App extends React.Component {
     error: PropTypes.object,
   };
 
+  getSnapshotBeforeUpdate() {
```

Comment:
What is the reason for using this hook? By the way, it lacks a `return null` and if you define this hook you have to define a `componentDidUpdate`.

In fact, if you simply replace `getSnapshotBeforeUpdate` with `componentDidUpdate` it works.